### PR TITLE
title full width on

### PR DESCRIPTION
**Pull request by pianeta051**
@nstjean

### CODE REVIEW

File: app/views/editor/rich.html.erb [hunk 1]
```diff
@@ -41,7 +41,7 @@
 
 <div class="pl-editor">
 
-  <div class="col-6 col-sm-12 ple-header">
+  <div class="col-12 ple-header">
```
Comment (nstjean) [inline app/views/editor/rich.html.erb]: This is perfect!

File: app/views/editor/rich.html.erb [hunk 2]
```diff
@@ -62,12 +62,12 @@
...
-          type="text" 
+        <input
+          class="form-control input-lg"
+          type="text"
           id="title-input"
-          placeholder="Title" 
-          value="<%= if @node then @node.title else params[:title] end %>" 
+          placeholder="Title"
+          value="<%= if @node then @node.title else params[:title] end %>"
         />
```
Comment (nstjean) [inline app/views/editor/rich.html.erb]: I'm not sure why it made all these changes to the rest of the lines?

Comment (cesswairimu) [reply app/views/editor/rich.html.erb]: Maybe its the text editor... I use vim and whenever I open a file with vim all the trailing white spaces are auto deleted...strange that the travis is complaining.. Restarting it... Thanks so much @nstjean and thanks a lot @pianeta051 for working on this

**Comment by nstjean** [comment]
The test passed, so this can be merged!

**Pull request merged**